src/settings_internal.js

```diff
@@ -175,7 +175,7 @@ var EXPECT_MAIN = 1;
 // MODULARIZE, and returned from the factory function.
 var EXPORT_READY_PROMISE = 1;
 
-var USE_LEGACY_DYNCALLS = 0;
+var USE_LEGACY_DYNCALLS = 1;
```

Comment:
I intended this setting to mean "use the dyncalls that are generated by binaryen".   Perhaps we don't need those add anymore after thise change?    In nay case can you add a comment here so its clear what the new meaning is?

Reply:
Hmm, that may be needed in some form or the other. Let me see what form the code takes after test suite passes.